### PR TITLE
DEV-612 Add new code field in sign up process

### PR DESCRIPTION
**Pull request by ahmedkamals**

### CODE REVIEW

File: api/invitations.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+categories: ["API"]
+title: Invitations
+---
+
+# Invitations
+
+## Check validity
+
+```sh
+$ curl {{ site.parku.api }}/invitations/ckdzUEdUcWs= \
```
Comment (christiangoltz) [inline api/invitations.md]: is that a working invitation code or just random stuff?

Comment (ahmedkamals) [inline api/invitations.md]: It is a working, I think I should change to a non-working one, correct?

Comment (christiangoltz) [inline api/invitations.md]: yes, this should be non-working, because otherwise users could use that to get a payback ;)

**Comment by christiangoltz** [comment]
:+1:

**Pull request merged**